src/styles/blocks/utils/mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin common-margins {
+  margin: 0 16px 16px;
+}
+
+@mixin box-sizing {
```

Comment:
delete it, you already added box-sizing in `*` for all elements, also delete box-sizing in other places